src/GraphExplorer.js

```diff
@@ -0,0 +1,313 @@
+/**
+ * Created by valeriy.abornyev on 11/9/2015.
+ */
+
+var vis = require('vis'),
+    $ = require('jquery'),
+    ElementsCollection = require('./ElementsCollection');
+
+
+    function GraphExplorer(settings) {
+        var visData;
+        var status = false;
+        var statusSuggestion = false;
+        var self = this;
+        this.container = settings.container;
+        this.options = settings.options;
+        this.inputData = {};
+
+
+        $.ajax({
+            method: 'get',
+            url: settings.data,
+            success: function (data) {
+                self.inputData = data.elements;
```

Comment:
What about encapsulating data fetching inside of ElementCollections ?
Is there are any obstacles for doing it ?